<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clicker</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QLCDNumber" name="laboratory_price">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>580</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="portal_number">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>550</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="click_number">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>310</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="main_click">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>Click</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="click_price">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>340</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="main_display">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>18</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="mine_number">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>470</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="factory_price">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>500</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="portal_price">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>580</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="farm_number">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>390</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="portal_push">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>550</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Portal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="click_push">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>click power</string>
     </property>
    </widget>
    <widget class="QPushButton" name="factory_push">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Factory</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="laboratory_number">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>550</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="grandma_price">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>420</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="factory_number">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>470</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="mine_push">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>470</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Mine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="grandma_push">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>grandma</string>
     </property>
    </widget>
    <widget class="QPushButton" name="laboratory_push">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Laboratory</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="grandma_number">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="farm_push">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Farm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Прокачки</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="mine_price">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>500</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="farm_price">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>420</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>6</number>
     </property>
    </widget>
    <widget class="QPushButton" name="save_push">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Как играть</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Leaderboard</string>
    </attribute>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>411</width>
       <height>601</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
